### PR TITLE
App would crash If avatar does not have springBones

### PR DESCRIPTION
**Pull request by tcm390**
check whether avatar has springBones before updating wind effect 

Issue:

https://user-images.githubusercontent.com/60634884/182854256-560b2270-e1c3-46e0-b484-3e5428a01a8d.mp4

Result:

https://user-images.githubusercontent.com/60634884/182854299-a44c88e4-96a0-41c7-a9b2-06863c9f0edc.mp4

### CODE REVIEW

File: avatars/avatars.js [hunk 1]
```diff
@@ -1981,7 +1981,7 @@ class Avatar {
     // update wind in simulation
     const _updateWind = () =>{
       const headPosition = localVector.setFromMatrixPosition(this.modelBoneOutputs.Head.matrixWorld);
-      wind.update(timestamp, headPosition, this.springBoneManager)
+      this.springBoneManager && wind.update(timestamp, headPosition, this.springBoneManager)
```
Comment (avaer) [inline avatars/avatars.js:1984]: Can you add a comment here to clarify that some avatars will not have spring bones?

**Review by avaer — CHANGES_REQUESTED** [review]
Recommending a comment.

File: avatars/avatars.js (hunk 1 above)
Comment (tcm390) [reply avatars/avatars.js:1985]: Added comment

https://github.com/webaverse/app/pull/3489/files

**Comment by alisaad673** [comment]
Tested it with the cyberpunk.vrm its working fine

**Review by avaer — APPROVED** [review]

**Pull request merged**